zerver/lib/markdown/__init__.py

```diff
@@ -1073,6 +1073,18 @@ def handle_tweet_inlining(
         if info['remove'] is not None:
             info['parent'].remove(info['remove'])
 
+    def handle_youtube_url_inlining(
+        self,
+        root: Element,
+        found_url: ResultWithFamily[Tuple[str, Optional[str]]],
+        youtube: str,
```

Comment:
What's bugging me about this code is that `youtube` is actually `youtube_thumbnail_image` or something, this variable name is misleading. Maybe we can use `yt_image` here keeping it in line with `yt_id`.